doc/install_centos6.txt

```diff
@@ -16,44 +16,16 @@ Easy Installation of an optimized Theano on CentOS 6
     investing money (i.e. buying a license to a proprietary BLAS
     implementation), it is possible to gain further performance.
```

Comment:
Ok I didn't know that! 
In that case I think we should remove this note and probably rethink the prerequisites section a little. I will comment there.